src/Misc/layoutbin/installdependencies.sh

```diff
@@ -36,7 +36,7 @@ then
         command -v apt
         if [ $? -eq 0 ]
         then
-            apt update && apt install -y liblttng-ust0 libcurl3 libkrb5-3 zlib1g
+            apt update && apt install -y liblttng-ust0 libcurl4 libkrb5-3 zlib1g
```

Comment:
will this affect Ubuntu 16.04/14.04 installation?

Reply:
https://packages.ubuntu.com/search?keywords=libcurl&searchon=names

Yes, libcurl3 is on 14.04, 16.04 and allegedly 18.04, but libcurl4 is only on 18.04
This would drop 14.04 and 16.04 and is therefore dangerous.

Given the docs say that libcurl3 should be available on ubuntu 18.04, I do wonder if there's a bug in their latest AMIs?

Reply:
Would it be worth putting this dependency check behind an apt-search? We could search libcurl4 first then try libcurl3 as fallback.
There are other packages that have similar problems too such as: https://github.com/Microsoft/azure-pipelines-agent/blob/master/src/Misc/layoutbin/installdependencies.sh#L58
and that case is solved by piping.